### PR TITLE
add-favicon: implictly include favicon at layout template

### PR DESCRIPTION
**Pull request by ivnnv**
Just adding the favicon as its not showing

### CODE REVIEW

File: src/server/views/layout.hbs [hunk 1]
```diff
@@ -22,6 +22,8 @@
     {{/if}}
     <link rel="stylesheet" href="{{ basePath }}/dashboard.css">
     <script>hljs.initHighlightingOnLoad();</script>
+    <!-- Favicon -->
+    <link rel="shortcut icon" href="{{ basePath }}/favicon.ico" type="image/x-icon">
```
Comment (roggervalf) [inline src/server/views/layout.hbs:26]: what about passing it as a new option like customCssPath, also this should be optional, only if customFavicon is provided:
```
{{#if customFavicon}}
    <link rel="shortcut icon" href="{{ customFavicon }}" type="image/x-icon">
{{/if}}
```

Comment (ivnnv) [reply src/server/views/layout.hbs:26]: Hi, I think I dont understand the comment, why having something like a custom favicon?
This PR intention is just showing the one that is already present at `/public/favicon.ico` but its not used.

Comment (roggervalf) [reply src/server/views/layout.hbs:26]: hey @ivnnv I tried your change locally, lgtm thank you

**Review by roggervalf — APPROVED** [review]

**Pull request merged**